src/Block/Service/AbstractClassificationBlockService.php

```diff
@@ -55,13 +55,14 @@ final protected function getFormAdminType(FormMapper $formMapper, AdminInterface
             'translation_domain' => 'SonataClassificationBundle',
         ], $adminOptions);
 
-        $fieldDescription = $admin->getModelManager()->getNewFieldDescriptionInstance($admin->getClass(), $field, $adminOptions);
+        $fieldDescription = $admin->getFieldDescriptionFactory()->create($admin->getClass(), $field, $adminOptions);
         $fieldDescription->setAssociationAdmin($admin);
         $fieldDescription->setAdmin($formMapper->getAdmin());
-        $fieldDescription->setAssociationMapping([
-            'fieldName' => $field,
-            'type' => ClassMetadataInfo::MANY_TO_ONE,
-        ]);
+        // TODO: Check how to replace the call to `setAssociationMapping()`.
```

Comment:
@sonata-project/contributors, do you know how this call can be replaced?

Reply:
I would say it's not needed, the persistence bundle should be able to do the job correctly.

Reply:
Ok, thank you.